core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/STDCMPostProcessing.kt

```diff
@@ -50,7 +50,8 @@ class STDCMPostProcessing(private val graph: STDCMGraph) {
         val blockRanges = makeBlockRanges(ranges)
         val blockWaypoints = makeBlockWaypoints(path)
         val chunkPath = makeChunkPathFromRanges(graph, ranges)
-        val trainPath = makePathProperties(infra!!, chunkPath)
+        val routes = ranges.lastOrNull()?.edge?.infraExplorer?.getExploredRoutes() ?: listOf()
```

Comment:
```suggestion
        val routes = ranges.last().edge.infraExplorer.getExploredRoutes()
```

Same, it can't be empty here